Update Readme.md (#2)
DevExpress-Examples/asp-net-web-forms-cardview-detail-data-in-popup-window: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T339685)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # Card View for ASP.NET Web Forms - How to display detail data in a popup window

Reviewer: ```suggestion
# Card View for ASP.NET Web Forms - How to display detail data in a pop-up window
```

<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/t339685/)**
<!-- run online end -->

In this example, [ASPxCardView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxCardView) contains a hyperlink column. 

```aspx
<dx:CardViewHyperLinkColumn FieldName="CustomerID" >
    <PropertiesHyperLinkEdit NavigateUrlFormatString="javascript:ShowDetailPopup('{0}');" />
</dx:CardViewHyperLinkColumn>
```

> When a user clicks a hyperlink, a popup window is opened to display detail data - orders of the current customer. The [SetContentUrl](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.SetContentUrl(url)) method specifies the web page ([Orders.aspx](./CS/Orders.aspx)) to be displayed in the window and the query paramenter (`customerID`) to select data from the data base.

Reviewer: ```suggestion
When a user clicks a hyperlink, a pop-up window opens that displays detail data: orders of the current customer. The [SetContentUrl](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.SetContentUrl(url)) method specifies the web page ([Orders.aspx](./CS/Orders.aspx)) to be displayed in the window and the query parameter (`customerID`) to select data from the database.
```
